src/server/server.rs

```diff
@@ -107,6 +110,9 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver + 'static> Server<T, S> {
             if let Some(engines) = debug_engines {
                 sb = sb.register_service(create_debug(DebugService::new(engines)));
             }
+            if let Some(service) = import_service {
```

Comment:
Is it possible `debug_engines` and `import_service` are both `Some(...)`?

Reply:
Yes, but they don't conflict with each other.